frontend/src/scenes/plugins/tabs/installed/InstalledTab.tsx

```diff
@@ -55,24 +143,86 @@ export function InstalledTab(): JSX.Element {
+                                    </Tag>
+                                </>
+                            ) : (
+                                `Enabled plugins (${enabledPlugins.length})`
+                            )
+                        }
+                        buttons={rearranging ? rearrangingButtons : upgradeButton}
+                    />
+                    {canRearrange || rearranging ? (
+                        <SortablePlugins useDragHandle onSortEnd={onSortEnd}>
+                            {enabledPlugins.map((plugin, index) => (
+                                <SortablePlugin
+                                    key={plugin.id}
+                                    plugin={plugin}
+                                    index={index}
+                                    order={index + 1}
+                                    maxOrder={enabledPlugins.length}
+                                    rearranging={rearranging}
+                                />
+                            ))}
+                        </SortablePlugins>
+                    ) : (
+                        <Row gutter={16} style={{ marginTop: 16 }}>
+                            {enabledPlugins.map((plugin, index) => (
+                                <InstalledPlugin
+                                    key={plugin.id}
+                                    plugin={plugin}
+                                    order={index + 1}
+                                    maxOrder={enabledPlugins.length}
+                                />
+                            ))}
+                        </Row>
+                    )}
+                </>
+            ) : null}
+
+            {disabledPlugins.length > 0 ? (
+                <>
+                    <Subtitle
+                        subtitle={`Installed plugins (${disabledPlugins.length})`}
+                        buttons={<>{enabledPlugins.length === 0 ? upgradeButton : null}</>}
+                    />
+                    <Row gutter={16} style={{ marginTop: 16 }}>
+                        {disabledPlugins.map((plugin) => (
+                            <InstalledPlugin key={plugin.id} plugin={plugin} />
+                        ))}
+                    </Row>
+                </>
+            ) : null}
+
+            {installedPlugins.length === 0 ? (
+                loading ? (
+                    <>
+                        <Subtitle subtitle="Enabled plugins" buttons={<Skeleton.Button style={{ width: 150 }} />} />
+                        <PluginLoading />
+                    </>
+                ) : (
                     <>
-                        {installedPlugins.map((plugin) => {
-                            return <InstalledPlugin key={plugin.id} plugin={plugin} />
-                        })}
-                        {installedPlugins.length == 0 && <Col span={24}>You don't have any plugins installed yet.</Col>}
+                        <Subtitle subtitle="Installed Plugins" />
```

Comment:
Looks like capitalization differs between Subtitles here.

Reply:
made the last one lowercase as well